javascript/organizing_your_javascript_code/json.md

```diff
@@ -4,16 +4,32 @@ JSON (JavaScript Object Notation) is a standardized format for structuring data.
 
 Fortunately, there isn't much to learn here. We're only including a lesson on it because some formatting rules can cause confusion if you aren't aware of them. Spend 10-15 minutes going through the following resources and you'll be good to go.
 
+### Lesson overview
```

Comment:
This file shouldn't have been updated. Can you revert these changes?

Reply:
@anasmustafa123 just checking that you've seen this comment as well, I see you resolved the above comment about duplicated content.

Reply:
@thatblindgeye  done, sorry again